packages/common/src/test/themeMount.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ReactElement } from 'react';
+import { mount } from 'enzyme';
+import { ThemeProvider } from 'styled-components';
+import theme from '../theme';
+
+const mountWithTheme = (tree: ReactElement<any>) => {
```

Comment:
I think we should try to avoid using `any` and use `unknown` instead.
https://mariusschulz.com/blog/the-unknown-type-in-typescript